badgecheck/tasks/utils.py

```diff
@@ -10,3 +10,14 @@ def task_result(success=True, message='', actions=None):
         actions = []
 
     return (success, message, actions,)
+
+
+def is_empty_list(value):
+    return isinstance(value, (tuple, list,)) and len(value) == 0
```

Comment:
Is there value in doing something like

```
import collections
...
return isinstance(value, collections.Sized) and len(value) == 0
```

Or one of the other ABCs in collections: `(collections.Sequence, collections.Set)`, for example?